superset-frontend/src/featureFlags.ts

```diff
@@ -48,8 +49,12 @@ declare global {
   }
 }
 
+let initialized = false;
 export function initFeatureFlags(featureFlags: FeatureFlagMap) {
-  window.featureFlags = featureFlags || {};
+  if (!initialized) {
```

Comment:
instead of adding an `initialized` variable, can we just check to see if `window.featureFlags` is set?